src/devices/Is31fl3730/FunctionRegister.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display.Internal
+{
+    /// <summary>
+    /// Register addresses for the Function Register.
+    /// </summary>
+    internal static class Is31fl3730_FunctionRegister
```

Comment:
You should not use underscore in the name of classes or enums.
also the file name should be the same as the class/enum.

Reply:
Right. Do you have a suggestion on better naming? The issue is this: https://github.com/dotnet/iot/blob/2580229760fd983b7ab6f0280ee2746cd4edca9c/src/devices/Is31fl3731/Registers.cs#L52

Reply:
I can go back and break that file into one per type.

Reply:
You can just remove the underscore and that would work. In all cases, rename as well the file so it's more coherent.